### DISCUSSION TITLE
OTA update

### DISCUSSION THREAD

**Discussion by marconr9** [huizebruin/s0tool#126] [General] [answered]
Hi,

Question how does the OTA update works  on the website of the sensor and which file do I need to upload.

**Comment by huizebruin** [comment]
Hello @marconr9 .

How do you mean?

To update the S0tool code inside esphome dashboard?

**Comment by marconr9** [comment, marked as answer]
Thanks,  i got it.

**Reply by huizebruin** [reply]
https://github.com/huizebruin/s0tool/issues/135#issuecomment-1826302056
